subsys/bluetooth/controller/Kconfig

```diff
@@ -179,7 +179,7 @@ config BT_CTLR_ISO_RX_BUFFERS
 	int "Number of Isochronous Rx buffers"
 	depends on BT_CTLR_SYNC_ISO || BT_CTLR_CONN_ISO
 	default 8
-	range 1 30
+	range 0 30
```

Comment:
BT_CTLR_ISO_TX_BUFFERS is set to default BT_ISO_TX_BUF_COUNT if BT_ISO. But when BT_ISO_TX_BUF_COUNT is 0, it will return error because BT_CTLR_ISO_TX_BUFFERS is out of the range

BT_CTLR_ISO_RX_BUFFERS, however, is independent of BT_ISO_RX_BUF_COUNT. I suggest changing the minimum values of the mentioned kconfigs to 0